core/01_core.js

```diff
@@ -364,6 +364,7 @@
     eventLoopHasMoreWork: () => ops.op_event_loop_has_more_work(),
     setPromiseRejectCallback: (fn) => ops.op_set_promise_reject_callback(fn),
     byteLength: (str) => ops.op_str_byte_length(str),
+    arrayBufferWasDetached: (ab) => ops.op_arraybuffer_was_detached(ab),
```

Comment:
Done! I added it because it was proposed by Andreu https://github.com/denoland/deno/pull/16307#discussion_r996438081

To be honest, I'm not sure what's the reason for exposing some methods to `Deno.core` if almost all of them are available in `Deno.core.ops`, couldn't find any info about the reasoning, AFAIK those APIs are not documented, although they're publicly exposed.

Reply:
It's legacy stuff, that was left to not break `deno_std`. We should really fix this...

Reply:
Good to know, thanks!